squad/frontend/templatetags/squad.py

```diff
@@ -91,6 +92,20 @@ def build_url(build):
     return reverse("build", args=(build.project.group.slug, build.project.slug, build.version))
 
 
+@register_global_function
+def previous_build_url(build):
+    prev_bld_status = ProjectStatus.objects.filter(build__project=build.project, build__datetime__lt=build.datetime).last()
```

Comment:
I chose to go thru the status here rather than the build just in case more info is needed.

Reply:
Any particular reason for `build__project=build.project` instead of `build=build`?

Reply:
oh wow, forgot am dealing with a db record already :man_facepalming: lol..thanks @chaw :)